packages/runtime/test/multiple-workers/basic.test.js

```diff
@@ -86,37 +86,6 @@ test('services are started with a single workers when no workers information is
   ok(!messages.includes('Starting the worker 0 of the service "composer"...'))
 })
 
-test('services are started with a single workers in development', async t => {
```

Comment:
why was this removed?

Reply:
This test is not valid anymore because the workers is not arbitrary set to one in non-production, as the change in `RoundRobinMap.configure`